migrations/20240128230553-create-limits-table.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const tableName = 'limits';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable(tableName, {
+      id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      label: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      type: {
+        type: Sequelize.ENUM('counter', 'boolean'),
+        allowNull: false,
+      },
+      value: {
+        type: Sequelize.INTEGER,
```

Comment:
This value is more flexible as a string. I got it, we can represent the boolean as 0,1, but in the hypothetic - and highly probable - case we need to set a string, this field would be useless. Leaving it as a string and then parsing the value depending on the type is more flexible in the long term.